dali/sasha/saxref.cpp

```diff
@@ -803,7 +814,11 @@ class CNewXRefManager: public CNewXRefManagerBase
         rawgrp->queryNode(0).endpoint().getHostText(gname.clear());
```

Comment:
I've no idea what this is for/doing..
I suspect we should at least put it in an if (!isContainerized())..
But can you check how it behaves in containerized setup and then discuss?

Also the cover above this.. it looks like it's scanning for other groups that contain the target group... 
Probably trying to cope with situations (on BM) where there are overlapping clusters.
If so need to be careful not to cause strange behaviour in a containerized setup, when most groups will be full of localhosts.. can you check what effect it has in a containerized setup with a couple of data planes?